cli/commands/migrate_status.go

```diff
@@ -74,12 +74,13 @@ func printStatus(status *migrate.Status) *bytes.Buffer {
 	buf := &bytes.Buffer{}
 	out.Init(buf, 0, 8, 2, ' ', 0)
 	w := util.NewPrefixWriter(out)
-	w.Write(util.LEVEL_0, "VERSION\tSOURCE STATUS\tDATABASE STATUS\n")
+	w.Write(util.LEVEL_0, "VERSION\tSOURCE STATUS\tDATABASE STATUS\tIDENTIFIER\n")
```

Comment:
@eulercamposbarros Can you change `IDENTIFIER` to `NAME`.

Also, change the order to `VERSION    NAME    SOURCE STATUS    DATABASE STATUS`

Reply:
Done!